modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/JobEndpoint.java

```diff
@@ -743,4 +787,18 @@ public int compare(Job job1, Job job2) {
       return ascending ? result : -1 * result;
     }
   }
+
+  /**
+   * @param hostname of server to find in list
+   */
+  private Optional<HostRegistration> findServerByHost(String hostname, List<HostRegistration> servers) {
+    // Java 1.8 return servers.stream().filter(o -> o.getBaseUrl().equals(hostname)).findFirst();
```

Comment:
You can use Java 8 in opencast/develop

Reply:
Yeah, I remembered last night I hadn't updated the code for java  8 (ported from our 3.7 java 7 repo)